tests/test_core.py

```diff
@@ -527,6 +527,11 @@ def test_no_units_for_info_enum(self):
         self.assertRaises(ValueError, Info, 'foo', 'help', unit="x")
         self.assertRaises(ValueError, Enum, 'foo', 'help', unit="x")
 
+    def test_name_cleanup_before_unit_append(self):
```

Comment:
Can we also add a test that this is exposed correctly somewhere?

Reply:
Added on tests/test_asgi.py and tests/test_wsgi.py

Reply:
Those are additional handlers, text_exposition would be the best place to put it.

Reply:
PTAL